cmd/template/cluster/provider/capa.go

```diff
@@ -120,6 +123,43 @@ func templateClusterAWS(ctx context.Context, k8sClient k8sclient.Interface, outp
 			},
 		}
 
+		if config.AWS.ClusterType == "proxy-private" {
+			c, _ := cidr.Parse(config.AWS.NetworkVPCCIDR)
+			subnets, err := c.SubNetting(cidr.MethodSubnetNum, 4)
```

Comment:
Can we add a comment why we split into 4?

Reply:
I dont know the reasoning behind it.

Reply:
@calvix @alex-dabija can we clarify this? Otherwise it will be hard to understand or change later.

Reply:
this should be the default splitting,  if no AZs are set,  we want to use 3 AZs which means we need 3 blocks for subnets (one is unused)

But we should not hardcode it, we should first check for the AZs and if they are set we should use that value (round up to an even number) and use that to split the VPC CIDR, if the AZs are not used then we use default split by 4

Reply:
right, as you would have trouble diving subnet into 6 equal CIDR blocks, it should be the closest bigger power of 2